docusaurus/docs/resources/native-form.md

```diff
@@ -29,23 +29,23 @@ nativeForm(spaceId[, params[, formAttributes][, type]]);
 
 ### Required params
 
--   spaceId: String
+- spaceId: String
 
 ### Optional params
 
--   params: Object. Additional parameters you want sent in the post.
--   formAttributes: Object. Set/override the form attributes like `target`, `method`, and `action`. `method` defaults to "post", `action` will default to "\`/ms/api/availity/internal/spc/magneto/sso/v1/saml/\${spaceId}\`", and `target` will default to "\_blank". Additional attributes can be defined and should be valid on an HTML form element.
--   type: String. Override the magneto integration type. Defaults to "saml"
+- params: Object. Additional parameters you want sent in the post.
+- formAttributes: Object. Set/override the form attributes like `target`, `method`, and `action`. `method` defaults to "post", `action` will default to "\`/ms/api/availity/internal/spc/magneto/sso/v1/saml/\${spaceId}\`", and `target` will default to "\_blank". Additional attributes can be defined and should be valid on an HTML form element.
+- type: String. Override the magneto integration type. Defaults to "saml"
 
 ###
 
 ```js
 import nativeForm from '@availity/native-form';
 
 nativeForm(
```

Comment:
`nativeForm` is now async so this should be `await nativeForm`
There's also the possibility of errors/promise rejections now that we are (potentially) fetching the proper sso type, so we should probably document this inside a try/catch with error handling.